addons/website/static/src/scss/website.scss

```diff
@@ -717,6 +717,12 @@ table.table_desc tr td {
     min-height: 10px;
 }
 
+// Gradient
+.gradient_text {
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
```

Comment:
Is not this the class defined by the odoo lib? Why is its related style defined in website?

Reply:
This class did not exist. I defined it in website, because I did not want it to appear in non-website pages.
Moving it to web_editor so that portal pages can use it as well.